### PR TITLE
Feature/automatic snapshot

### PR DESCRIPTION
**Pull request by DoctorVoid**

### CODE REVIEW

File: test/integration/server/schema/resolvers.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import { PolarisGraphQLContext } from '@enigmatis/polaris-common';
-import { PubSub } from 'apollo-server-express';
-import { DeleteResult, getPolarisConnectionManager, Like } from '../../../../src/index';
-import { TestContext } from '../context/test-context';
-import { Author } from '../dal/entities/author';
-import { Book } from '../dal/entities/book';
-import { polarisGraphQLLogger } from '../utils/logger';
+import {PolarisGraphQLContext} from '@enigmatis/polaris-common';
```
Comment (osher-sade) [inline test/integration/server/schema/resolvers.ts]: Not prettified, execute `npm run lint:fix` and push the prettified code

File: test/integration/server/utils/snapshot-client.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+import * as polarisProperties from "../resources/polaris-properties.json";
+
+export const url = `http://localhost:${polarisProperties.port}/snapshot`;
```
Comment (osher-sade) [inline test/integration/server/utils/snapshot-client.ts]: If you are just using the port, import it directly, like this:
`import { port } from "../resources/polaris-properties.json";`

**Review by osher-sade — APPROVED** [review]

File: test/integration/server/schema/resolvers.ts (hunk 1 above)
Comment (DoctorVoid) [reply test/integration/server/schema/resolvers.ts]: OK PARTUSHKA

**Pull request merged**